src/modules/ekf2/EKF2.cpp

```diff
@@ -1035,7 +1060,7 @@ void EKF2::PublishSensorBias(const hrt_abstime &timestamp)
 			accel_bias.copyTo(bias.accel_bias);
 			bias.accel_bias_limit = _params->acc_bias_lim;
 			_ekf.getAccelBiasVariance().copyTo(bias.accel_bias_variance);
-			bias.accel_bias_valid = !_ekf.fault_status_flags().bad_acc_bias;
+			bias.accel_bias_valid = !_ekf.fault_status_flags().bad_acc_bias && _acc_cal_available;
```

Comment:
Note there's still the high rate `vehicle_acceleration` topic (analogous to `vehicle_angular_velocity`) that grabs the accel bias from the primary estimator and applies it directly (unless invalid).

Reply:
Yes, the control loops using acceleration should have the benefits of the latest bias estimate.